src/cmd/cli/command/commands.go

```diff
@@ -507,205 +512,101 @@ var certGenerateCmd = &cobra.Command{
 	},
 }
 
+func afterGenerate(ctx context.Context, result setup.SetupResult) {
+	term.Info("Code generated successfully in folder", result.Folder)
+	editor := pkg.Getenv("EDITOR", "code")
```

Comment:
This is a regression: terminal-mode editors like `vim` will run in the background without a TTY attached. Fix: https://github.com/DefangLabs/defang/pull/1370